### PR TITLE
add describeApps methods

### PR DESCRIPTION
**Pull request by appmode**
required for https://github.com/easybib/issues/issues/5008

The existing getAllApps() and getAllAppsForStack() methods return only the app name & id, the new describe methods return the entire app description array.

### CODE REVIEW

File: src/EasyBib/Service/Opsworks/Opsworks.php [hunk 1]
```diff
@@ -278,6 +278,47 @@ public function getAllApps()
...
     /**
+     * Returns all apps belonging to a stack
+     *
+     * @param string $stackId Opsworks Stack Id
+     *
+     * @return array
+     */
+    public function describeAllAppsForStack($stackId)
+    {
+        $opsworksApps = array();
```
Comment (till) [inline src/EasyBib/Service/Opsworks/Opsworks.php]: `$opsworksApps = [];`

Comment (till) [inline src/EasyBib/Service/Opsworks/Opsworks.php]: And so on. ;) Short array syntax.

**Comment by till** [comment]
@appmode minor nitpick! But otherwise :+1: :+1: :+1:

File: src/EasyBib/Service/Opsworks/Opsworks.php (hunk 1 above)
Comment (appmode) [inline src/EasyBib/Service/Opsworks/Opsworks.php]: yeah, that's a copy-paste from the original method I was duplicating, have replaced them everywhere now.

**Pull request merged**